src/proof/range_proof.rs

```diff
@@ -345,4 +346,29 @@ mod tests {
         assert!(result.is_ok());
     }
 
+    #[bench]
+    fn bench_range_proof(b: &mut Bencher){
+        // TODO: bench range for 256bit range.
+        b.iter(|| {
+            /// common:
+            let range = BigInt::from(0xFFFFFFFFFFFFFi64);
```

Comment:
Where is this big int from? In other words, why that value?

Reply:
just a small number I used for testing. I replaced it now with a 256bit number to make a more realistic scenario and benchmarks